### PR TITLE
Basic carousel functioning

### PR DESCRIPTION
**Pull request by logandudley**

### CODE REVIEW

**Comment by logandudley** [comment]
@dj12589 Hey Daria, can you please review this PR?

File: client/app.jsx [hunk 1]
```diff
@@ -29,17 +31,24 @@ class App extends React.Component {
...
+    const images = this.state.imageUrls.map(imageUrl => {
+      return (
+      <div className='imgDiv'>
+        <img src={imageUrl} alt="shirt" />
+      </div>
+      )
+    });
+
+    const settings = {
+      dots: false
```
Comment (dj12589) [inline client/app.jsx]: Can you change dots to something else, just for clearer variable naming?

File: client/app.jsx [hunk 2]
```diff
@@ -2,15 +2,17 @@ import React from 'react';
 import ReactDom from 'react-dom';
```
Comment (dj12589) [inline client/app.jsx]: Nit: may want to change to ReactDOM.

File: client/styles.css [hunk 3]
```diff
@@ -0,0 +1,13 @@
+.container {
+  padding: 40px;
+  /* background: #419be0; */
```
Comment (dj12589) [inline client/styles.css]: Do you want to take out this comment?

Comment (logandudley) [reply client/styles.css]: Yes I do.

File: client/app.jsx (hunk 1 above)
Comment (logandudley) [reply client/app.jsx]: "dots" is a prop name used by the react-slick library.

File: client/app.jsx (hunk 2 above)
Comment (logandudley) [reply client/app.jsx]: Will do.

**Pull request merged**